packages/atomic/package.json

```diff
@@ -61,6 +62,7 @@
     "jest": "^26.4.2",
     "jest-cli": "^26.4.2",
     "postcss-import": "^14.0.0",
+    "puppeteer": "^9.1.1",
```

Comment:
The removal of pupeteer, although useless, caused random errors to appear in the dev build 
<img width="1151" alt="Screen Shot 2021-05-05 at 2 45 37 PM" src="https://user-images.githubusercontent.com/4923043/117198392-fae69280-adb6-11eb-84dc-05ab59604ecf.png">